R/lazyframe-frame.R

```diff
@@ -2485,4 +2485,63 @@ lazyframe__sink_parquet <- function(
   })
 }
 
+#' Evaluate the query in streaming mode and write to an IPC file
+#'
+#' @inherit lazyframe__sink_parquet description params return
+#' @inheritParams rlang::args_dots_empty
+#' @param compression Must be one of:
+#' * `"lz4"`: fast compression/decompression.
+#' * `"zstd"`: good compression performance.
+#'
+#' If `NULL`, it uses `"zstd"`.
```

Comment:
I think this interpretation is incorrect: the official JS implementation of Apache Arrow cannot read compressed Arrow files. Therefore, uncompressed Arrow files are important.
If `NULL` is interpreted as `"zstd"`, it means that there is no way to write an uncompressed Arrow file.

See also pola-rs/polars#15644

Reply:
I was going to fix this as it is complicated to explain, but I didn't have time today.

Reply:
In short, the mismatch with the Parquet writer on the Rust side requires a different strategy.

Reply:
I moved the function and tests to dedicated files so the comments here are marked outdated but are still relevant.

I'm still puzzled by this, upstream polars uses allows `"uncompressed"` on the python side but this hits the `must be one of {'zstd', 'lz4'}` error when passed to Rust. Can you finish this PR since you seem to know what to do?